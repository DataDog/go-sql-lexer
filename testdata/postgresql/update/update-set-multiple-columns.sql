UPDATE users SET name = 'Jane Updated', email = 'jane.updated@example.com' WHERE id = 2;
