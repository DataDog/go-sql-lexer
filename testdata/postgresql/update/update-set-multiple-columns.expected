[
    {
        "expected": "UPDATE users SET name = ?, email = ? WHERE id = ?",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "UPDATE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]